main/resource-access.bs

```diff
@@ -156,6 +156,25 @@ requests.
 [[Source](https://github.com/solid/specification/pull/160#issuecomment-636822687)].
 
 
+## Constraints and Problem Details ## {#constraints-problem-details}
+
+Servers MUST publish any constraints on clients’ ability to create or update
```

Comment:
Are authorization policies in a WAC acl file a constraint?

Reply:
No, this excludes the details of authorization policies straight off an ACL. It'd be more like linking to the constraint that for example says that server does not allow client to update containment triples.

Reply:
Assuming that shape validation would fall into this constraint bucket, it might be helpful to use that as a concrete example?

Reply:
Do you mean to use the URI of a Shape Validation resource as example? Possibly, but I think I prefer to use a URI that will exist for sure as example, like a URI from the spec. That doesn't rule out the possibility to use server-specific constraints possibly with SV. Just thought that it may be more clear.

I'm still interested in documentation that clarifies the differences between solid:shape and ldp:constrainedBy - beyond https://gitter.im/solid/data-interoperability-panel?at=5e398f03f6945f41ef5ad6a6 and what's currently in the draft spec.

Can we come back to the example after this PR? If an example is desired, we can refer to a requirement in the spec along the lines of:

```
Link: <https://solid.github.io/specification/#http-client>; rel="http://www.w3.org/ns/ldp#constrainedBy"
```

but actually have a URI for the statement and use that in the example:

>A Solid client MUST use the Content-Type HTTP header in PUT, POST and PATCH requests [RFC7231].

Reply:
It may be worthwhile to distinguish between the abilities that all clients need to fulfil and the abilities that a (particular) client is expected to fulfil for against a particular resource. Both are important. If solid:shape is already advertising the requirement (or thereof), I'm not sure what the added benefit would be to duplicate that target URI for the ldp:constrainedBy relation. So, from that point, I'd lean on well-defined abilities that a client needs to fulfil to create or update. solid:shape is specific to data whereas ldp:constrainedBy can be for anything (including data).